zerver/lib/create_user.py

```diff
@@ -89,6 +89,7 @@ def create_user_profile(
     timezone: str,
     default_language: str = "en",
     tutorial_status: str = UserProfile.TUTORIAL_WAITING,
+    email_address_visibility: Optional[int] = UserProfile.EMAIL_ADDRESS_VISIBILITY_EVERYONE,
```

Comment:
This is an `int`, not `Optional`, right?

Reply:
Yes, updated the type to be `int` and also made it mandatory kwarg.